learntools/ml_intermediate/ex6.py

```diff
@@ -50,7 +71,7 @@ def check(self, predictions_1):
         assert len(predictions_1) == 292, \
         "Please generate predictions on the validation data."
         
-        assert round(predictions_1[0]) == 237696, \
+        assert round(predictions_1[0]) == round(basic_predictions[0]), \
         ("Are you sure that you used the training data to train the model?" 
```

Comment:
I think there are other things that would cause them to get the wrong prediction. This error message might be making too many assumptions.